### PR TITLE
Changed course and lesson error messages

### PR DESCRIPTION
**Pull request by OlyaKorchan**
- changed error messages for course and lesson
- edited setAlert to make multiline alerts more readable 

<details>
<summary>New Сourse</summary>

https://github.com/ita-social-projects/SpaceToStudy-Client/assets/17857767/5871e757-c562-4556-aca6-7bd4ac27905f

</details>

<details>
<summary>New Lesson</summary>

https://github.com/ita-social-projects/SpaceToStudy-Client/assets/17857767/df791efc-83d3-4faa-8da1-4298c29b57ba

</details>

### CODE REVIEW

File: src/context/snackbar-context.tsx [hunk 1]
```diff
@@ -62,7 +62,11 @@ export const SnackBarProvider = ({ children }: SnackBarProviderProps) => {
           sx={{ color: 'basic.white' }}
           variant='filled'
         >
-          {t(message)}
+          {t(message)
+            .split(', ')
+            .map((line) => (
+              <div key={line}>{line}</div>
```
[the remark this answers is not in the document]
Comment (OlyaKorchan) [reply src/context/snackbar-context.tsx]: > Can you use Box insted of div ?

changed to Box)

**Comment by dmtrth25** [comment]
Can you fix that ? 

image

I think the message will be enough for the error.
For example: `The title field cannot be empty`
What do you think ?

**Review by dmtrth25 — COMMENTED** [review]
Can you fix that ? 

image

I think the message will be enough for the error.
For example: `The title field cannot be empty`
What do you think ?

**Review by MariaAfonina — APPROVED** [review]

**Review by Mav-Ivan — APPROVED** [review]

File: src/pages/create-course/CreateCourse.tsx [hunk 2]
```diff
@@ -44,7 +44,7 @@ const CreateCourse = () => {
   const onResponseError = (error: ErrorResponse) => {
     setAlert({
       severity: snackbarVariants.error,
-      message: error ? `errors.${error.code}` : ''
+      message: error ? `errors.${error.message}` : ''
```
Comment (Tolik170) [inline src/pages/create-course/CreateCourse.tsx:47]: how it will be work for other errors for which we using translation with err.code?

**Comment by OlyaKorchan** [comment]
> Can you fix that ?
> 
> image
> I think the message will be enough for the error. For example: `The title field cannot be empty` What do you think ?

discussed on the meeting
that is the index of the section, that is received from backend

**Review by maarekua — APPROVED** [review]

**Pull request merged**